src/service/loot.ts

```diff
@@ -400,6 +399,15 @@ const lootTemplates: loot.LootTemplate[] = [
         emote: ":newspaper2:",
         asset: "assets/loot/31-bravo.jpg",
     },
+    {
+        id: LootTypeId.VERSCHIMMELTER_DOENER,
+        weight: 0, // cannot be obtained via drop
```

Comment:
Könnten ne Konstante einführen um das deutlich zu machen. Clean Code und so

Reply:
Gelöst in 9d238624959c82fe4a2ca89256686f47730250cf